Assets/Scripts/RandomBlockCreator.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RandomBlockCreator : MonoBehaviour
+{
+    // to indicate block in current scene (in Inspector)
+    // and parent block, which will always be "Blocks" empty object
+    [SerializeField] GameObject blockObjType;
+    [SerializeField] GameObject parentBlock;
+
+    // our project uses fixed 4-3 ratio, so these world units will not be changing
+    float screenWidthWorld = 16f;
+    // float screenHeightWorld = 12f; // not used, but keeping just in case
+
+    // for random block settings - max and min widths
+    [SerializeField] int minBlockWidthWorld = 1;
+    [SerializeField] int maxBlockWidthWorld = 4;
+
+    // for random block settings - start row to place blocks
+    [Range(3,7)]
+    [SerializeField] float blockStartRow = 5f;
+
+    // starting block width
+    [SerializeField] int prefabBlockWidthWorld = 1;
+
+    [SerializeField] float spaceBetweenBlocks = 0.1f;
+    [SerializeField] float blockHeightScale = 0.5f;
+
+    private bool isBlockFR(){
+        return blockObjType.name == "block_fr";
```

Comment:
On our basic starter scene, this line gives the following error: "The variable blockObjType of RandomBlockCreator has not been assigned. You probably need to assign the blockObjType variable of the RandomBlockCreator script in the inspector." I think the best way to handle this is to set the RandomBlockCreator of the Player Supervisor prefab's Block Obj Type to Block.

Reply:
Ah - good catch. I updated that and just pushed another commit to this repo. Hopefully that fixes it!